contrib/Dockerfile.test

```diff
@@ -18,17 +18,16 @@ RUN apk add --no-cache $PACKAGES && \
   make install
 
 # Final image
-FROM alpine:edge
```

Comment:
Hey @pratikbin makes sense for us to refresh these. Not a big fan of `edge` or `latest` though. it may be a good idea to lock these down to a specific version, WDYT?

Reply:
That's okay too if you want to manually upgrade alpine versions. I'll revert it back.